website/scripts/build-static-content.js

```diff
@@ -48,6 +55,41 @@ module.exports = {
         if (queries.length !== _.uniq(_.pluck(queries, 'slug')).length) {
           throw new Error('Failed parsing YAML for query library: Queries as currently named would result in colliding (duplicate) slugs.  To resolve, rename the queries whose names are too similar.  Note the duplicates: ' + _.pluck(queries, 'slug').sort());
         }//•
+        // Report any errors that were detected along the way in one fell swoop to avoid endless resubmitting of PRs.
+        if (queriesWithProblematicContributors.length >= 1) {
+          throw new Error('Failed parsing YAML for query library: The "contributors" of a query should be a single string of valid GitHub user names (e.g. "zwass", or "zwass,noahtalerman,mikermcneil").  But one or more queries have an invalid "contributors" value: ' + _.pluck(queriesWithProblematicContributors, 'slug').sort());
+        }//•
+
+        // Get a distinct list of all GitHub usernames from all of our queries.
+        // Map all queries to build a list of unique contributor names then build a dictionary of user profile information from the GitHub Users API
+        const githubUsernames = queries.reduce((list, query) => {
+          if (!queriesWithProblematicContributors.find((element) => element.slug === query.slug)) {
+            list = _.union(list, query.contributors.split(','));
+          }
+          return list;
+        }, []);
+
+        // Talk to GitHub and get additional information about each contributor.
+        let githubDataByUsername = {};
+        await sails.helpers.flow.simultaneouslyForEach(githubUsernames, async(username)=>{
+          githubDataByUsername[username] = await sails.helpers.http.get('https://api.github.com/users/' + encodeURIComponent(username), {}, { 'User-Agent': 'Awesome-Octocat-App', Accept: 'application/vnd.github.v3+json' });
```

Comment:
@mikermcneil, I think we still need to update the GitHub API call with FleetDM token and change the `User-Agent` header to something more appropriate for Fleet 🐻🐱?

Reply:
Good point.  Maybe something with “Fleet” in it.  Not an advertisement; just something somewhat accurate.

Tip: to set request headers, you can do .get.with() to put your HTTP call into “named parameter” mode, so then you can pass in:
```
await sails.helpers.http.get.with({
  url: '...',
  headers: { ‘user-agent’: '...' }
})
```

Reply:
(in my experiences, header names seem to be parsed in a way that’s case insensitive on the server side, so I’d say it’s pretty likely GitHub’s servers do the same. If one was curious, one could try changing the case on the api token header to see if that’s true.  Or better yet we could just use the casing from their docs so that this doesn’t change on us someday.  I’m just being nerdy)